.buildkite/pipeline.yml

```
@@ -203,3 +203,16 @@ steps:
           image: "rustvmm/dev:v6"
           always-pull: true
           propagate-environment: true
+
+- label: "cargo-audit"
+     commands:
+       - cargo audit
```

```suggestion
       - cargo audit -q
```
This is what I was suggesting in #36 (sorry if it wasn't clear enough). Do you have any concerns about this change?